### DISCUSSION TITLE
react-email and iOS are not compatible yet

### DISCUSSION THREAD

**Discussion by SECHUBSYSTEM** [resend/react-email#1477] [Q&A]
Hello guys, so I'm a full stack software engineer who's recently working on an email based project, I'm using react.email and postmark, however my styles and paddings aren't showing up right on all iOS I've checked, my mails look perfect on other OS but looks terrible on iOS. I'm referring to the gmail and apple iOS email clients. Please does any one have a solution or has anyone faced something similar and was able to resolve it?
![WhatsApp Image 2024-05-22 at 20 07 25_e5a0930c](https://github.com/resend/react-email/assets/106301523/bf242f79-9626-4e0d-be6a-4931091d640e)
![WhatsApp Image 2024-05-22 at 20 04 15_3f5055c9](https://github.com/resend/react-email/assets/106301523/d2fe72ac-fd42-4fdf-a97a-d4a3ba098e52)
![WhatsApp Image 2024-05-22 at 20 05 11_192e53df](https://github.com/resend/react-email/assets/106301523/45d90571-e458-4ed8-98b7-f29c7f326995)

**Comment by gabrielmfern** [comment]
Let's keep the discussion for this in a single place, so just #1478.

**Comment by SECHUBSYSTEM** [comment]
Wasn't a duplicate, I needed answers both as a discussion and an
issue...you shouldn't have closed it!
